src/test/java/context/healthinformatics/Parser/XMLParserTest.java

```diff
@@ -35,22 +35,27 @@ public void before() {
 	 */
 	@Test
 	public void correctTest() {
+		final int startLine = 7;
+		final int col1 = 2;
+		final int col2 = 5;
+		final int col3 = 6;

```

Comment:
That doesnt make any sense. We would like  to be able to test different files/ settings per test. So global would be bad.